torch/_decomp/__init__.py

```diff
@@ -48,6 +48,24 @@
 meta_table = global_decomposition_table["meta"]
 
 
+def _should_decompose_because_unsafe_op(op: torch._ops.OperatorBase) -> bool:
+    """
+    Returns True if the op must always decompose in export/compile tracing system
+
+    In export, we always decompose certain CIA ops that are tagged with
+    maybe_aliasing_or_mutating because we statically need to know if the op is
+    mutating or not. But these CIA ops could have different behaviour in runtime.
+
+    native_batch_norm is a prim op which has a wrong schema and it needs to be replaced
+    with correct schema. But until then, we will force decompose it
+    """
+    if not isinstance(op, torch._ops.OpOverload):
+        return False
+    if torch.Tag.maybe_aliasing_or_mutating in op.tags:
```

Comment:
Can we just give this tag to `native_batch_norm` directly so it doesn't need to be special-cased here?

Reply:
I think the problem here is native_batch_norm is not a CIA op so basically its schema is just wrong.